packages/runtime/src/components/page/Page.tsx

```diff
@@ -237,6 +237,167 @@ export function Page({ document: page }: Props): JSX.Element {
   )
 }
 
+type PropsV1 = { pageData: MakeswiftPageData }
+
+export function PageV1({ pageData }: PropsV1): JSX.Element {
```

Comment:
Just had an idea - let's not `export` this type so that the code can truly remain latent, but we can move forward developing snapshotting.

That's if we decide to merge. If we keep a long running branch separate from `main` and just regularly rebase, then there's no need for that.

I'm ok with either.